docs/topics/basic-syntax.md

```diff
@@ -133,44 +133,87 @@ See [Functions](functions.md).
 
 ## Variables
 
-Read-only local variables are defined using the keyword `val`. They can be assigned a value only once.
+In Kotlin, you declare a variable starting with a keyword, `val` or `var`, followed by the name of the variable.
+
+Use the `val` keyword to declare variables that are assigned a value only once. These are read-only local variables that can’t be reassigned a different value
+after initialization: 
 
 ```kotlin
 fun main() {
 //sampleStart
-    val a: Int = 1  // immediate assignment
-    val b = 2   // `Int` type is inferred
-    val c: Int  // Type required when no initializer is provided
-    c = 3       // deferred assignment
+    // Declares the variable x and initializes it with the value of 5
+    val x: Int = 5
+    // 5
 //sampleEnd
-    println("a = $a, b = $b, c = $c")
+    println(x)
 }
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-basic-syntax-val"}
 
-Variables that can be reassigned use the `var` keyword.
+Use the `var` keyword to declare variables that can be reassigned. These are mutable variables, and you can assign other values to them after initialization:
```

Comment:
I would suggest wording it in a way to have the word reassign/change the variable to belabor further the point that these variables can be changed, especially that we use the word reassigns in the code example